### DISCUSSION TITLE
Is there a release plan?

### DISCUSSION THREAD

**Discussion by indoorhill** [MrPowers/chispa#67] [General]
Hi MrPowers!

Is there a process/timeline for releases? There have been a few PRs that have been merged this year, but there has not been a release in quite some time.

**Comment by indoorhill** [comment]
@MrPowers, any comments/updates you can share?

**Comment by MrPowers** [comment]
@ethnhll - I will do a release now, thanks for the ping.  It'd be good for us to setup milestones with due dates, so the releases can be more predictable.  Thanks again for following up.

**Comment by MrPowers** [comment]
I published v0.9.3: https://pypi.org/project/chispa/

**Reply by indoorhill** [reply]
thank you! :D
